### PR TITLE
sync recent updates

### PR DESCRIPTION
**Pull request by Jinjiang**
... to #bdf00bb

Diff:
https://github.com/vuejs/vuejs.org/compare/7adb0e4...bdf00bb
but except:
* #9c8262b Why Vue video for the Front Page (#1325)
* #8e63cbb abort vimeo modal init if vimeo not in page

I think these 2 commits should be considered later because it has a Vimeo video which is blocked here.

Thanks.

### CODE REVIEW

File: themes/vue/source/js/common.js [hunk 1]
```diff
@@ -193,6 +201,7 @@
       } else {
         headers = content.querySelectorAll('h3')
         each.call(headers, function (h) {
+          console.log(h)
```
Comment (Justineo) [inline themes/vue/source/js/common.js]: 删了？

File: src/v2/guide/team.md [hunk 2]
```diff
@@ -591,12 +591,25 @@ order: 803
...
       },
       reposPersonal: [
-        'monterail/vue-multiselect', 'monterail/vue-newsletter'
+        'shentao/vue-multiselect'
+      ]
+    },
+    {
+      name: 'kingwl',
+      title: 'New Bee',
+      city: 'Beijin, China',
```
[the remark this answers is not in the document]
Comment (Jinjiang) [reply src/v2/guide/team.md]: 地理位置都没翻译，而且这个还会在“找附近的团队成员”功能中用到 (需要和变量 `cityCoordsFor` 中的字段匹配) 我觉得翻译的必要不大了，不太影响理解。

[the remark this answers is not in the document]
Comment (Justineo) [reply src/v2/guide/team.md]: Beijing 少了一个 g

[the remark this answers is not in the document]
Comment (Jinjiang) [reply src/v2/guide/team.md]: Oops，才发现 `cityCoordsFor` 里还没有北京，还要提供经纬度，才能正常工作。目测这个地方不只是一个拼写问题。

**Comment by Jinjiang** [comment]
Done
Btw. 事实上 `pointers-event: none` 之后那个 find near by 按钮一直是点下去没有反应的状态🤷‍♂️

**Pull request merged**